src/rdkafka_conf.c

```diff
@@ -471,6 +471,16 @@ static const struct rd_kafka_property rd_kafka_properties[] = {
 	  _RK(ssl.crl_location),
 	  "Path to CRL for verifying broker's certificate validity."
 	},
+	/* Andrea Minuto (PKCS12) */
+	{ _RK_GLOBAL, "ssl.pkcs12.keystore.location", _RK_C_STR,
```

Comment:
We'll need to synchronize these property names with the Java client.
Ping @ijuma